boxsdk/object/api_json_object.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+from collections import Mapping
+from abc import ABCMeta
+import six
+
+from .base_api_json_object import BaseAPIJSONObject, BaseAPIJSONObjectMeta
+
+
+class APIJSONObjectMeta(BaseAPIJSONObjectMeta, ABCMeta):
```

Comment:
Thnx for explaining... I think that's worth a link & comment.
http://code.activestate.com/recipes/204197-solving-the-metaclass-conflict/
or something saying this addresses the known issue with inheriting from 2 classes with separate metaclasses.